src/cljs/rems/application.cljs

```diff
@@ -74,26 +57,23 @@
   [id & [replace?]]
   (dispatch! (str "#/application/" id) replace?))
 
-(defn- format-validation-messages
-  [application msgs]
-  (let [fields-by-id (index-by [:id] (map localize-item (:items application)))
-        licenses-by-id (index-by [:id] (map localize-item (:licenses application)))]
+(defn- format-validation-errors
+  [application errors]
+  (let [fields-by-id (->> (get-in application [:application/form :form/fields])
+                          (index-by [:field/id]))
+        licenses-by-id (->> (get-in application [:application/licenses])
+                            (index-by [:license/id]))]
     [:div (text :t.form/validation.errors)
      (into [:ul]
            (concat
-            (for [{:keys [type field-id]} (filter :field-id msgs)]
-              [:li (text-format type (:title (fields-by-id field-id)))])
-            (for [{:keys [type license-id]} (filter :license-id msgs)]
-              [:li (text-format type (:title (licenses-by-id license-id)))])))]))
-
-
-
-
-
-
-
-
-
+            (for [{:keys [type field-id]} (filter :field-id errors)]
```

Comment:
or perhaps
```
(for [{:keys [type field-id]} errors
      :when field-id]
   ...)

Reply:
or perhaps factor out a `format-validation-error` function that can handle field and license errors?